app/src/main/res/layout/card_infobox.xml

```diff
@@ -19,25 +19,26 @@
 	<LinearLayout
 		android:layout_width="match_parent"
 		android:layout_height="wrap_content"
+		android:clipChildren="false"
+		android:clipToPadding="false"
 		android:orientation="horizontal"
 		android:paddingStart="@dimen/spacing_medium_large"
-		android:paddingTop="@dimen/spacing_medium_large"
-		android:paddingEnd="@dimen/spacing_medium_large"
-		android:paddingBottom="@dimen/spacing_large"
-		android:clipChildren="false"
-		android:clipToPadding="false">
+		android:paddingBottom="@dimen/spacing_large">
 
 		<ImageView
 			android:id="@+id/contacs_chevron"
 			android:layout_width="@dimen/icon_size"
 			android:layout_height="@dimen/icon_size"
-			android:src="@drawable/ic_info"
+			android:layout_marginTop="@dimen/spacing_medium_large"
 			android:layout_marginRight="@dimen/spacing_medium"
+			android:src="@drawable/ic_info"
 			android:tint="@color/white" />
 
 		<LinearLayout
-			android:layout_width="match_parent"
+			android:layout_width="0dp"
```

Comment:
I think the X button should only take away space from title, but not the text and the link.